### DISCUSSION TITLE
Any plans to get 'Verified creator' badge?

### DISCUSSION THREAD

**Discussion by MSauchanka** [artilleryio/artillery#2547] [Artillery]
Hey guys,

Since my company would like to use actions from 'Verified creator` only I have a question. Any plans for artillery to get verified?

**Comment by hassy** [comment]
No plans at the moment @MSauchanka. The docs are unclear on the process - https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#about-badges-in-github-marketplace
